{{cookiecutter.repo_name}}/.travis.yml

```diff
@@ -13,7 +13,7 @@ addons:
 install:
 # Upgrade pip setuptools and wheel
 - pip install -U pip wheel setuptools
-- pip install --no-use-pep517 .[precommit,testing]
+- pip install .
```

Comment:
the app doesn't have a `setup.py`, so does this do anything?

Can I suggest you add this step also in the top-level .travis.yml, similar to what is done in the plugin cutter https://github.com/aiidateam/aiida-plugin-cutter/blob/master/.travis.yml#L59 ?

Reply:
Done.

The update to the top-level `.travis.yml` file can be found in the diff.